### PR TITLE
Fix Table Sorting

### LINKED ISSUE #19: Shiny rate wrongly sorted by string

### LINKED ISSUE #20: Raid level 6 not shown as mega

### PR DESCRIPTION
**Pull request by versx**
Fix: #19 
Fix: #20 

Fixes table sorting for the following:
- Shiny stats
- Raids
- Gyms
- Invasions

### CODE REVIEW

**Comment by versx** [comment]
Anyone test yet?

**Review by clburlison — APPROVED** [review]
👍

File: src/data/map.js [hunk 1]
```diff
@@ -458,12 +467,27 @@ async function getRaids(filter) {
...
+                        raid_level: {
+                            formatted: parseInt(level) === 6 ? 'Mega' : 'Level ' + level,
+                            sort: level
+                        },
                         gym_name: `<a href='${mapLink}' target='_blank'>${gym}</a>`,
-                        team: teamIcon,
+                        team: {
+                            formatted: teamIcon,
+                            sort: row.team_id
+                        },
```
Comment (Mygod) [inline src/data/map.js:490]: This seems unnecessary but ok.

Comment (versx) [reply src/data/map.js:490]: Oh I'm sure someone will complain that teams aren't sorted by 0, 1, 2, 3 so I think it is.

**Pull request merged**